packages/gatsby-plugin-netlify-cms/README.md

```diff
@@ -178,6 +178,13 @@ bar).
 Customize the value of the `favicon` tag in your CMS HTML (shows in the browser
 bar).
 
+### `excludeRobots`
```

Comment:
Let's flip this over to `includeRobots` since this is opt-in and by default it is _not_ included

Reply:
And done